testsuite/cluster-test/src/cluster.rs

```diff
@@ -86,7 +99,8 @@ impl Cluster {
             prometheus_ip.ok_or_else(|| format_err!("Prometheus was not found in workplace"))?;
         Ok(Self {
             instances,
-            prometheus_ip,
+            prometheus_ip: Some(prometheus_ip),
+            mint_file: String::from(mint_file),
```

Comment:
Most common pattern in our codebase is to use `.to_string()` to convert from &str to String